scss/_navbar.scss

```diff
@@ -138,9 +134,11 @@
   width: 1.5em;
   height: 1.5em;
   vertical-align: middle;
-  content: "";
-  background: no-repeat center center;
-  background-size: 100% 100%;
+  // Background shorthand for:
+  // background-position: center
+  // background-size: 100%
+  // background-repeat: no-repeat
```

Comment:
I don't think the comment is needed. I mean, if we are going this way, we'd need to add a comment in every shorthand.